drivers/hmis/app/graphql/mutations/submit_form.rb

```diff
@@ -44,7 +44,6 @@ def _resolve(input:, record_lock_version: nil)
       raise HmisErrors::ApiError, 'Record not found' unless record.present?
 
       # Check permission
-      allowed = nil
```

Comment:
why is this removed?

Reply:
oh, because my IDE was telling me "Local variable 'allowed' is not used after assignment". but I am happy to re-add it.

Reply:
ohh k.. no that's true